### PR TITLE
Program GUI Ver2

### PR DESCRIPTION
**Pull request by TechyScientist**
Update for ProgramGUI to create a dummy pos system

### CODE REVIEW

File: LoginGUI/src/ProgramGUI.java [hunk 1]
```diff
@@ -35,27 +42,42 @@
...
+	private final int SELF = 0;
+	private final int ALL = 1;
+	
 	private static final int FRAME_WIDTH = 300;
 	private static final int FRAME_HEIGHT = 145;
+	private static int scope;
 	
 	//Private nonstatic JComponents for access in this class
 	private JTextField user;
 	private JTextField pass;
```
Comment (pagzamaphone) [inline LoginGUI/src/ProgramGUI.java]: you probably want the pass to be a passwordField https://docs.oracle.com/javase/tutorial/uiswing/components/passwordfield.html

File: LoginGUI/src/LoginGUI.java [hunk 2]
```diff
@@ -142,12 +142,14 @@ public void actionPerformed(ActionEvent e) {
...
+						new ProgramGUI("admin", logged);
 					}
 					else if(users[i].equals(user.getText() + ":" + pass.getText() + ";reg")) {
+						logged = user.getText();
 						LoginGUI.this.dispose();
-						new ProgramGUI("reg");
+						new ProgramGUI("reg", logged);
 					}
 				}
 			}
```
Comment (pagzamaphone) [inline LoginGUI/src/LoginGUI.java]: i'd like to see a little more feedback to the user if they enter an invalid username/password, right now, if you click the "confirm" button on the login page and you didn't enter a valid input it appears like the button does nothing

File: LoginGUI/src/ProgramGUI.java [hunk 3]
```diff
@@ -129,17 +155,18 @@ else if(cmd.getText().toLowerCase().equals("deluser")) {
...
 			if(type.equals("admin")) {
-				rename();
+				rename(ALL);
 			}
 			else {
-				error(PERMISSIONS);
+				rename(SELF);
 			}
 		}
 		//promote user command
```
Comment (pagzamaphone) [inline LoginGUI/src/ProgramGUI.java]: promote / demote should have an "user exists" check prior to promoting / demoting a user, I was able to promote / demote a user that did not exist

**Comment by pagzamaphone** [comment]
Great start! I love seeing projects like this. I would love to see some more work on this before I review it again, I gave some feedback that should lead you down some paths of code to add here.

**Comment by TechyScientist** [comment]
Thanks @pagzamaphone! I will work on it some more and fix these issues.

**Comment by TechyScientist** [comment]
@pagzamaphone, There is a user check on promote and demote that seems to work for me

**Comment by pagzamaphone** [comment]
+1

**Pull request merged**